src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -103,8 +112,48 @@ protected internal override PinMode GetPinMode(int pinNumber)
 
         protected internal override bool IsPinModeSupported(int pinNumber, PinMode mode)
         {
-            // Libgpiod Api do not support pull up or pull down resistors for now.
-            return mode != PinMode.InputPullDown && mode != PinMode.InputPullUp;
+            switch (mode)
+            {
+                case PinMode.Input:
+                    return true;
+                case PinMode.InputPullDown:
+                case PinMode.InputPullUp:
+                    // for use the bias flags we need libgpiod version 1.5 or later
+                    IntPtr libgpiodVersionPtr = Interop.libgpiod.gpiod_version_string();
+                    var libgpiodVersionMatch = Marshal.PtrToStringAnsi(libgpiodVersionPtr).Trim().Split('.');
```

Comment:
Is Trim actually needed? Can https://docs.microsoft.com/en-us/dotnet/api/system.version.-ctor?view=netframework-4.8#System_Version__ctor_System_String_ be used instead?

Reply:
Thank you for the review @krwq. Thank you very much for the tip, this simplified the code. I am sending a new commit using `Version`, let me know what you think.